components/search/MoreFiltersDropdownFacet.vue

```diff
@@ -111,6 +113,17 @@
         this.$nextTick(() => {
           this.$emit('selectedOptions', this.name, this.selectedOptions);
         });
+      },
+
+      toggleShowMore() {
+        this.isActive = !this.isActive;
+
+        this.$nextTick(() => {
+          let index;
+          this.isActive ? index = this.limitTo : index = this.limitTo - 1;
```

Comment:
Replace these two lines with:
```js
const index = this.limitTo - (this.isActive ? 0 : 1);
```
This has lower cognitive complexity, and `index` is never mutated so should use `const`.